src/Microsoft.Azure.WebJobs.Storage/StorageExceptionExtensions.cs

```diff
@@ -815,5 +815,31 @@ public static string GetErrorCode(this StorageException exception)
 
             return extendedInformation.ErrorCode;
         }
+
+        /// <summary>
+        /// Returns a custom detailed error message for a StorageException. This is a workaround for bad error messages
+        /// returned by Azure Storage.
+        /// </summary>
+        /// <param name="exception">The storage exception.</param>
+        /// <returns>The error message.</returns>
+        public static string GetDetailedErrorMessage(this StorageException exception)
+        {
+            if (exception == null)
+            {
+                throw new ArgumentNullException("exception");
+            }
+
+            string message = exception.Message;
+
+            if (exception.RequestInformation != null)
+            {
+                message += String.Format(" (HTTP status code {0}: {1}. {2})",
```

Comment:
Rather than using `string.Format`, we've started using string interpolation whenever we add new strings (if we remember). It makes it a little easier to read. So can you change this to something like `$" (HTTP status code {exception.RequestInformation.HttpStatusCode}: {exception.RequestInformation.ExtendedErrorInformation?.ErrorCode}. {exception.RequestInformation.ExtendedErrorInformation?.ErrorMessage})"`?

Reply:
Thanks!!